### PR TITLE
feat: create upload component

### PR DESCRIPTION
**Pull request by ValMgr**
close: #70

### CODE REVIEW

File: src/lib/components/UploadImage.svelte [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+		/>
+		<svg width="64" height="64" viewBox="0 0 64 64" fill="none" xmlns="http://www.w3.org/2000/svg">
+			<path
+				d="M7.11111 28.8L14.2222 21.6889L33.7778 41.2444L46.2222 28.8L56.8889 39.4667V7.11111H7.11111V28.8ZM7.11111 38.8587V56.8889H18.1333L28.7502 46.2756L14.2222 31.7476L7.11111 38.8587ZM28.192 56.8889H56.8889V49.5253L46.2222 38.8587L28.192 56.8889ZM3.55556 0H60.4444C61.3874 0 62.2918 0.374602 62.9586 1.0414C63.6254 1.70819 64 2.61256 64 3.55556V60.4444C64 61.3874 63.6254 62.2918 62.9586 62.9586C62.2918 63.6254 61.3874 64 60.4444 64H3.55556C2.61256 64 1.70819 63.6254 1.0414 62.9586C0.374602 62.2918 0 61.3874 0 60.4444V3.55556C0 2.61256 0.374602 1.70819 1.0414 1.0414C1.70819 0.374602 2.61256 0 3.55556 0ZM44.4444 24.8889C43.03 24.8889 41.6734 24.327 40.6732 23.3268C39.673 22.3266 39.1111 20.97 39.1111 19.5556C39.1111 18.1411 39.673 16.7845 40.6732 15.7843C41.6734 14.7841 43.03 14.2222 44.4444 14.2222C45.8589 14.2222 47.2155 14.7841 48.2157 15.7843C49.2159 16.7845 49.7778 18.1411 49.7778 19.5556C49.7778 20.97 49.2159 22.3266 48.2157 23.3268C47.2155 24.327 45.8589 24.8889 44.4444 24.8889Z"
+				fill="black"
+			/>
+		</svg>
+		<h3>Téleversez un fichier</h3>
```
Comment (FlorianVDP) [inline src/lib/components/UploadImage.svelte:44]: Don't need h3, isn't good for SEO

**Review by FlorianVDP — APPROVED** [review]
I approve, nothing blocking the feature, but be careful for the h3, Marketing will create an issue for this

File: src/lib/components/UploadImage.svelte [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+			}
+
+			src = URL.createObjectURL(input.files[0]);
+		}
+	};
+</script>
+
+{#if type === 'classic'}
+	<div class="container-upload-img">
+		<input
```
[the remark this answers is not in the document]
Comment (ValMgr) [reply src/lib/components/UploadImage.svelte:30]: Isn't the job of this component. Label would be managed by the form himself

[the remark this answers is not in the document]
Comment (FlorianVDP) [reply src/lib/components/UploadImage.svelte:30]: Ok 👍🏼

**Pull request merged**